### DISCUSSION TITLE
iPXE on esxi

### DISCUSSION THREAD

**Discussion by PatcoreShare** [ipxe/ipxe#432] [Q&A] [answered]
Hi, 
I have vmware esxi in my structure. 
I Create Virtual machine and configure like this:  https://github.com/ipxe/ipxe/discussions/305
Booting via iPXE works, but installation took 4 days! 

At the end, such an error popped up

![błąd iscsi](https://user-images.githubusercontent.com/11398415/128667377-c424cb7e-7759-4790-9024-f4d17d7384e4.png)

Why the installation is so slow? 
How can I speed up the installation?
Where should I look for a problem? Any suggestions

Thanks for help

**Comment by NiKiZe** [comment, marked as answer]
Did you add `set netX/gateway 0.0.0.0`?
See https://forum.ipxe.org/showthread.php?tid=5948 under:
"After booting the Windows installer with a SAN drive hooked, my SAN drive isn't appearing in disk manager, or it is very slow. How can I fix this?"

Have you tried to use tcpdump or wireshark on you iSCSI server to see if there is any issues with the transfers?

**Reply by PatcoreShare** [reply]
Thank you very much, it helped!
